src/components/Book.js

```diff
@@ -12,28 +13,36 @@ const Book = ({ data, id }) => {
   };
   return (
     <section className="book-card">
-      <div>
+      <div className="book-info">
         <h1>{category}</h1>
         <h2>{title}</h2>
         <h3>{author}</h3>
-        <div>
+        <div className="book-action-button">
           <button type="button">Comments</button>
+          <div className="bar-verti" />
           <button
             type="button"
             onClick={handleRemoveBook}
           >
             Remove
           </button>
+          <div className="bar-verti" />
           <button type="button">Edit</button>
         </div>
       </div>
 
-      <div>
-        <p>0%</p>
-        <p>Completed</p>
+      <div className="book-progress">
+        <AiOutlineLoading3Quarters className="progress-bar" size="4rem" color="#0290ff" />
+        <div className="progress-text">
+          <p className="percent-complete">
+            {Math.floor(Math.random(100) * 100)}
+            %
+          </p>
+          <p className="complete">Completed</p>
+        </div>
       </div>
```

Comment:
- [ ] @Matthieu96Code great work on your design, however, the progress bar color doesn't match the design given:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/230712083-5d5bde89-29c7-48ae-aa89-2dd9bff56ca0.png)  | ![image](https://user-images.githubusercontent.com/61408860/230712126-548e83a0-2597-40d3-98e4-6579fce8a4f0.png)